src/pyhf/infer/mle.py

```diff
@@ -22,6 +22,10 @@ def fit(data, pdf, init_pars=None, par_bounds=None, **kwargs):
     """
     Run a unconstrained maximum likelihood fit.
 
+    .. note::
+
+        :func:`twice_nll` is the objective function.
```

Comment:
I would have noted that this is what is returned when `return_fitted_val=True` is passed, but I can follow up on this in another PR.